cmd/main.go

```diff
@@ -86,7 +84,7 @@ func main() {
 	flag.StringVar(&storePath, "store-path", ".", "The path to store publisher and subscription info.")
 	flag.StringVar(&transportHost, "transport-host", "http://ptp-event-publisher-service-NODE_NAME.openshift-ptp.svc.cluster.local:9043", "The transport bus hostname or service name.")
 	flag.IntVar(&apiPort, "api-port", 8089, "The address the rest api endpoint binds to.")
-	flag.StringVar(&apiVersion, "api-version", "1.0", "The address the rest api endpoint binds to.")
+	flag.StringVar(&apiVersion, "api-version", "2.0", "The address the rest api endpoint binds to.")
 	flag.StringVar(&httpEventPublisher, "http-event-publishers", "", "Comma separated address of the publishers available.")
```

Comment:
httpEventPublisher is no longer needed. It was used for the sidecar on consumer side, but with v2 there is no longer sidecar on consumer side.